fda-model/paireddata/PairedData.cs

```diff
@@ -120,6 +120,47 @@ public bool IsMonotonicallyIncreasing(double[] arrayOfData)
             return new PairedData(x, y);
         }
 
+        public IPairedData SumYsForGivenX(IPairedData input)
```

Comment:
please add xml comments - what is this for?

Reply:
is it not clear that this function sums the y values for a given x value? 

we use this to compute the total stage-damage function - summing the damage (y value) over the damage categories for a given stage.

Reply:
instead of just summing the ead from each?